provider/pkg/gen/typegen.go

```diff
@@ -202,7 +202,9 @@ func (d definition) isTopLevel() bool {
 	// by Pulumi.
 	switch fmt.Sprintf("%s/%s", d.gvk.GroupVersion().String(), d.gvk.Kind) {
 	case "policy/v1beta1/Eviction", "policy/v1/Eviction", "v1/Status", "apps/v1beta1/Scale", "apps/v1beta2/Scale",
-		"autoscaling/v1/Scale", "extensions/v1beta1/Scale", "core/v1/ComponentStatus", "core/v1/ComponentStatusList":
+		"autoscaling/v1/Scale", "extensions/v1beta1/Scale", "core/v1/ComponentStatus", "core/v1/ComponentStatusList",
+		"authentication/v1/TokenRequest", "authentication/v1/TokenRequestPatch", "authentication/v1/TokenReview",
+		"authentication/v1/TokenReviewPatch":
```

Comment:
It looks like there may be some related resources that should also be removed.

`authentication/v1beta1/TokenReview` and possibly all versions of `SelfSubjectReview` if that is another imperative command.

Reply:
Ah, good catch. Removed the other api versions of TokenReview. Have also removed SelfSubjectReview as that is also an imperative API call.